src/ext-test/java/org/opentripplanner/ext/fares/impl/OrcaFareServiceTest.java

```diff
@@ -362,23 +349,11 @@ void calculateFareForSounderLeg() {
     List<Leg> rides = List.of(
       getLeg(SOUND_TRANSIT_AGENCY_ID, "S Line", 0, "King Street Station", "Auburn Station")
     );
-    calculateFare(rides, regular, usDollars(4.25f));
-    calculateFare(rides, FareType.senior, usDollars(4.25f));
-    calculateFare(rides, FareType.youth, Money.ZERO_USD);
-    calculateFare(rides, FareType.electronicSpecial, ORCA_SPECIAL_FARE);
-    calculateFare(rides, FareType.electronicRegular, usDollars(4.25f));
-    calculateFare(rides, FareType.electronicSenior, ONE_DOLLAR);
-    calculateFare(rides, FareType.electronicYouth, Money.ZERO_USD);
-    // Ensure that it works in reverse
-    rides =
-      List.of(
-        getLeg(SOUND_TRANSIT_AGENCY_ID, "N Line", 0, "King Street Station", "Everett Station")
-      );
-    calculateFare(rides, regular, usDollars(5.00f));
-    calculateFare(rides, FareType.senior, usDollars(5.00f));
+    calculateFare(rides, regular, DEFAULT_TEST_RIDE_PRICE);
```

Comment:
This sequence is now the same as the one starting at line 338, so perhaps they could be combined.

Reply:
I added both legs to the same test. I think that still tests the same stuff now that the fares are the same.